### ISSUE TITLE
KML test files are bundled in library release

### ISSUE THREAD

**Issue by petrdvorak** [googlemaps/android-maps-utils#463]
```
Android Studio 3.1.3
Build #AI-173.4819257, built on June 4, 2018
JRE: 1.8.0_152-release-1024-b01 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13.6
```

**Summary:** 

I am very probably missing something (and I apologize for my ignorance in advance). However, I didn't find any guidelines on this in the [Setup Guide](https://developers.google.com/maps/documentation/android-sdk/utility/setup).

I created a new vanilla app in a fairly new Android Studio with the minimum and target SDK set on 27. Then, I added the following two dependencies to my module `gradle.properties` file:

```rb
implementation 'com.google.android.gms:play-services-maps:15.0.1'
implementation 'com.google.maps.android:android-maps-utils:0.5'
```

After I successfully built the app, I can see the following files in the resulting `*.apk` file:

```
amu_ballon_gx_prefix.kml
amu_basic_folder.kml
amu_basic_placemark.kml
amu_cdata.kml
amu_default_balloon.kml
amu_document_nest.kml
amu_draw_order_ground_overlay.kml
amu_extended_data.kml
amu_ground_overlay_color.kml
amu_ground_overlay.kml
amu_inline_style.kml
amu_multigeometry_placemarks.kml
amu_multiple_placemarks.kml
amu_nested_folders.kml
amu_nested_multigeometry.kml
amu_poly_style_boolean_alpha.kml
amu_poly_style_boolean_numeric.kml
amu_unknwown_folder.kml
amu_unsupported.kml
amu_visibility_ground_overlay.kml
```

Could the documentation mention a way to exclude these unnecessary resources? For example, while I see the benefit of using the Android Maps Utils in our banking app in general, I do not see any benefit of having an `amu_ballon_gx_prefix.kml` file packaged in the app as well, with the following example content:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<kml xmlns="http://www.opengis.net/kml/2.2"
 xmlns:gx="http://www.google.com/kml/ext/2.2">

  <Document>
    <name>balloonVisibility Example</name>
    <open>1</open>
      <Style id="linestyleExample">
        <LineStyle>
          <color>7f0000ff</color>
          <width>4</width>
          <gx:labelVisibility>1</gx:labelVisibility>
        </LineStyle>
      </Style>
    <gx:Tour>
      <name>Play me</name>
      <gx:Playlist>

        <gx:FlyTo>
          <gx:duration>8.0</gx:duration>
          <gx:flyToMode>bounce</gx:flyToMode>
          <LookAt>
            <longitude>-119.748584</longitude>
            <latitude>33.736266</latitude>
            <altitude>0</altitude>
            <heading>-9.295926</heading>
            <tilt>84.0957450</tilt>
            <range>4469.850414</range>
            <gx:altitudeMode>relativeToSeaFloor</gx:altitudeMode>
          </LookAt>
        </gx:FlyTo>

        <gx:AnimatedUpdate>
          <gx:duration>0.0</gx:duration>
          <Update>
            <targetHref/>
            <Change>
              <Placemark targetId="underwater1">
                <gx:balloonVisibility>1</gx:balloonVisibility>
              </Placemark>
            </Change>
          </Update>
        </gx:AnimatedUpdate>

        <gx:Wait>
          <gx:duration>4.0</gx:duration>
        </gx:Wait>

        <gx:AnimatedUpdate>
          <gx:duration>0.0</gx:duration>
          <Update>
            <targetHref/>
            <Change>
              <Placemark targetId="underwater1">
                <gx:balloonVisibility>0</gx:balloonVisibility>
              </Placemark>
            </Change>
          </Update>
        </gx:AnimatedUpdate>

        <gx:FlyTo>
          <gx:duration>3</gx:duration>
          <gx:flyToMode>smooth</gx:flyToMode>
          <LookAt>
            <longitude>-119.782630</longitude>
            <latitude>33.862855</latitude>
            <altitude>0</altitude>
            <heading>-9.314858</heading>
            <tilt>84.117317</tilt>
            <range>6792.665540</range>
            <gx:altitudeMode>relativeToSeaFloor</gx:altitudeMode>
          </LookAt>
        </gx:FlyTo>

        <gx:AnimatedUpdate>
          <gx:duration>0.0</gx:duration>
          <Update>
            <targetHref/>
            <Change>
              <Placemark targetId="underwater2">
                <gx:balloonVisibility>1</gx:balloonVisibility>
              </Placemark>
            </Change>
          </Update>
        </gx:AnimatedUpdate>

        <gx:Wait>
          <gx:duration>4.0</gx:duration>
        </gx:Wait>

        <gx:AnimatedUpdate>
          <gx:duration>0.0</gx:duration>
          <Update>
            <targetHref/>
            <Change>
              <Placemark targetId="underwater2">
                <gx:balloonVisibility>0</gx:balloonVisibility>
              </Placemark>
            </Change>
          </Update>
        </gx:AnimatedUpdate>

        <gx:FlyTo>
          <gx:duration>3</gx:duration>
          <gx:flyToMode>smooth</gx:flyToMode>
          <LookAt>
            <longitude>-119.849578</longitude>
            <latitude>33.968515</latitude>
            <altitude>0</altitude>
            <heading>-173.948935</heading>
            <tilt>23.063392</tilt>
            <range>3733.666023</range>
            <altitudeMode>relativeToGround</altitudeMode>
          </LookAt>
        </gx:FlyTo>

        <gx:AnimatedUpdate>
          <gx:duration>0.0</gx:duration>
          <Update>
            <targetHref/>
            <Change>
              <Placemark targetId="onland">
                <gx:balloonVisibility>1</gx:balloonVisibility>
              </Placemark>
            </Change>
          </Update>
        </gx:AnimatedUpdate>

        <gx:Wait>
          <gx:duration>4.0</gx:duration>
        </gx:Wait>

      </gx:Playlist>
    </gx:Tour>

    <Placemark id="underwater1">
      <name>Underwater off the California Coast</name>
      <description>
        The tour begins near the Santa Cruz Canyon,
        off the coast of California, USA.
      </description>
      <Point>
        <gx:altitudeMode>clampToSeaFloor</gx:altitudeMode>
        <coordinates>-119.749531,33.715059,0</coordinates>
      </Point>
    </Placemark>

    <Placemark id="underwater2">
      <name>Still swimming...</name>
      <description>We're about to leave the ocean, and visit the coast...</description>
      <Point>
        <gx:altitudeMode>clampToSeaFloor</gx:altitudeMode>
        <coordinates>-119.779550,33.829268,0</coordinates>
      </Point>
    </Placemark>

    <Placemark id="onland">
      <name>The end</name>
      <description>
        <![CDATA[The end of our simple tour.
        Use <gx:balloonVisibility>1</gx:balloonVisibility>
        to show description balloons.]]>
      </description>
      <Point>
        <coordinates>-119.849578,33.968515,0</coordinates>
      </Point>
    </Placemark>


  </Document>
</kml>
```

**Comment by barbeau** [comment]
@petrdvorak Thanks for flagging this.  Looks like these KML files were added to the library resources in https://github.com/googlemaps/android-maps-utils/pull/155, and from a quick look I'm not convinced they were included in the right place.

It looks like these KML files are used in the KML unit tests in `library/tests/src/com/google/maps/android/kml/Kml*Test.java`, and that they were mistakenly added to the `library/res/raw/` resource directory instead of the `...test...` resource directory.  Because of this, they get bundled in the library release.

So, I believe the correct solution for this is to move the KML test files to a test resource directory, where they are accessible by the library unit tests but they don't get compiled into the release library.

**Comment by petrdvorak** [comment]
Thank you very much for the elaboration, Sean. I agree with the suggestion to move the resources to test folder. Will you have a look, or should I try to fix this and prepare a PR?

**Comment by barbeau** [comment]
PRs are great - my spare time is pretty slim these days :).

**Comment by jpoehnelt** [comment]
closed by #529

**Comment by jpoehnelt** [comment]
This will be available in 0.6.1 shortly.
